### PR TITLE
CXP-1698: Add meta-schema 1.0.0 and update links

### PR DESCRIPTION
**Pull request by BenjaminRbt**
https://akeneo.atlassian.net/browse/CXP-1679

**Creates the v1.0.0 of the product mapping schema
Remove the BETA tag on the Catalogs for Apps feature
Add what's new blocks for prices mono and multi-currencies, multi-locales and variation management**

- [x] REST API - Catalogs endpoints
- [ ] **REST API - Mapped product endpoint (update the x-example in app_catalogs_id_mapped_products.yaml)**
- [x] REST API - Mapping schema for products
- [x] Catalogs for apps - update the jsonschemavalidator link
- [x] Catalogs for apps - upload the new metaschema file + update the metaschema link
- [x] Catalog for apps - update JSON schema example

### CODE REVIEW

**Comment by akeneo-secops** [comment]
### Gitleaks PR check
🛑 **We detected secrets within this PR**<br/>If this detection is a **true positive**, please rotate the secret **ASAP**.<br/><br/>If this secret is a false positive, you can add the corresponding fingerprint<br/>to your <code>.gitleaksignore</code> file and commit the change to this branch. More details in [Notion](https://www.notion.so/akeneo/Prevent-secrets-leakage-cad5e759ccbd436cb4a7966db3203e94).<br/><br/><details>- [Fingerprint faea0332dc977c1e5dbd926980456ac1a5c29c85:content/swagger/akeneo-web-api.json:generic-api-key:1](https://github.com/akeneo/pim-api-docs/blob/faea0332dc977c1e5dbd926980456ac1a5c29c85/content/swagger/akeneo-web-api.json#L1)</br>- [Fingerprint faea0332dc977c1e5dbd926980456ac1a5c29c85:content/swagger/akeneo-web-api.json:generic-api-key:1](https://github.com/akeneo/pim-api-docs/blob/faea0332dc977c1e5dbd926980456ac1a5c29c85/content/swagger/akeneo-web-api.json#L1)</br></details>

File: content/news/2023.md [hunk 1]
```diff
@@ -54,14 +54,44 @@ More info...
 -->
 
 <!-- Months always are title level 2 with ## before -->
+## December 2023
```
Comment (tomglvng) [inline content/news/2023.md:57]: January 2024?

Comment (BenjaminRbt) [reply content/news/2023.md:57]: I thought about this too, but I would also like to have opinion of @RiffVentura

File: content/apps/catalogs.md [hunk 2]
```diff
@@ -112,155 +113,45 @@ Once you have an enabled catalog, paginate the related products using this [endp
...
-    },
-    "name": {
-      "title": "Product name",
-      "type": "string"
-    },
-    "body_html": {
-      "title": "Description (textarea)",
-      "description": "Product description in raw HTML",
-      "type": "string",
-      "minLength": 0,
```
Comment (eoliveauakeneo) [inline content/apps/catalogs.md:176]: Here we have on the old example some constraints, we are missing those in the new example file.

**Comment by tomglvng** [comment]
@BenjaminRbt , can you also update `content/files/Akeneo PIM API.postman_collection.json`? I think it will be useful for new users

File: content/news/2023.md [hunk 3]
```diff
@@ -54,14 +54,44 @@ More info...
...
+
 ## November 2023
 
+### <span class="label label-news label-added">ADDED</span> Retrieve prices using Catalogs for Apps
+
+We updated our product mapping meta-schema.  
+Your app can now ask for price objects in one specific currency or multi-currencies.
+
+👉 Learn more about [how to update your catalog to define managed currencies](/api-reference.html#patch_app_catalog) and [how to define your product mapping schema](/apps/catalogs.html)
+
```
Comment (RiffVentura) [inline content/news/2023.md:88]: Exactly same paragraph  as above,  maybe we can remove it ?

File: content/news/2023.md [hunk 4]
```diff
@@ -99,31 +129,31 @@ Bootstrap a draft App in no time with our example that integrates now the OpenID
...
 
 ### <span class="label label-news label-added">ADDED</span> Mapping for Apps using catalogs
 
-Using the Catalog feature, you provide your users with a way to create a product selection quickly. 
-And now, with the mapping schema, you send all information the PIM needs to display a mapping UI to your users. 
+Using the Catalog feature, you provide your users with a way to create a product selection quickly.
+And now, with the mapping schema, you send all information the PIM needs to display a mapping UI to your users.
 
-Thanks to the mapping configuration, the PIM flats products to send them in the right structure to your app, 
+Thanks to the mapping configuration, the PIM flats products to send them in the right structure to your app,
```
[the remark this answers is not in the document]
Comment (RiffVentura) [reply content/news/2023.md:142]: past news. let's not change those

**Review by hogren — APPROVED** [review]

**Comment by BenjaminRbt** [comment]
Hello @hogren ,

Do you know if the team accountable for this PR is yours?

**Pull request merged**